scheduler/src/cook/kubernetes/api.clj

```diff
@@ -800,7 +800,10 @@
         params-env (build-params-env parameters)
         progress-env (task/build-executor-environment job)
         checkpoint-env (checkpoint->env checkpoint)
-        main-env-base (merge environment params-env progress-env sandbox-env checkpoint-env)
+        metadata-env (cond-> {"COOK_COMPUTE_CLUSTER_NAME" compute-cluster-name
+                              "COOK_SCHEDULER_REST_URL" (config/scheduler-rest-url)}
+                       pool-name (assoc "COOK_POOL" pool-name))
```

Comment:
Will this be `nil` if the user doesn't specify a pool (and lands in the default pool)? I assume we want to make it resolve to the name of the default pool?